Libraries/Storage/AsyncStorage.js

```diff
@@ -223,15 +224,15 @@ const AsyncStorage = {
   multiGet: function(
     keys: Array<string>,
     callback?: ?(errors: ?Array<Error>, result: ?Array<Array<string>>) => void,
-  ): Promise {
+  ): Promise<?Array<Array<string>>> {
     if (!this._immediate) {
       this._immediate = setImmediate(() => {
         this._immediate = null;
         this.flushGetRequests();
       });
     }
 
-    const getRequest = {
+    const getRequest: Object = {
```

Comment:
Is this annotation necessary?

Reply:
The annotation is necessary to pass flow. I'll define a more specific type.